### DISCUSSION TITLE
Version 1.4.202 seems to want access to \Device\HarddiskVolume3 and \Device\HarddiskVolumeShadowCopy3. Why?

### DISCUSSION THREAD

**Discussion by cridus** [nefarius/HidHide#136] [Q&A] [answered]
As the title says, version 1.4.202 wants to access `\Device\HarddiskVolume3` and `\Device\HarddiskVolumeShadowCopy3`. Windows defender prevented the access automatically.  
Why does HidHide need access to these paths now?

**Comment by nefarius** [comment]
The driver code hasn't changed last 3 releases, no idea where that comes from.

**Reply by cridus** [reply]
All I know is that while updating HidHide I got these two warnings:

App or process blocked: msiexec.exe
Protected folder: \Device\HarddiskVolume3

and

App or process blocked: SrTasks.exe
Protected folder: \Device\HarddiskVolumeShadowCopy3

and the installation failed.

**Reply by nefarius** [reply, marked as answer]
Both `msiexec.exe` (Windows Installer) and `SrTasks.exe` (System Restore Point manager) are core Windows components I have nothing to do with. From what you described it looks like a system restore point was (attempted) to be created - which is quite normal for MSI to trigger that - and that was somehow flagged as malicious. In essence still has nothing to do with my software but the Windows Installer procedure so IDK how to help you.

**Reply by cridus** [reply]
Then I don't know why the installation failed. Welp.
